scripts/katib_yaml/katib-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
Need to double-check whether `kubeflow` namespace is needed, or we can just skip it and use default one.